<launch>
	<node pkg="perfect_velodyne" type="normal_estimation_hdl32" name="NormalEstimationForVelodyne" output="screen" />
		<param name="HNN" value="3l" />
		<param name="MAX_RANGE" value="120.0f" />
		<param name="MIN_RANGE" value="2.2f" />
		<param name="VR" value="0.985f" />			<!--20151104更新-->
		<param name="skip" value="5.0f" />
		<param name="vector_horizon" value="0.1f" />
		<param name="vector_vertical" value="1.5f" />
		<param name="DENS" value="0.5f" />
		<param name="CURV" value="0.1f" />
		<param name="DISP" value="0l" />
</launch>


